ShadowDOM/explainer.md

```diff
@@ -83,7 +83,118 @@ This document explores several proposals that would allow developers to apply st
 ## Non-goals
 Some developers have expressed interest in CSS selectors crossing through the Shadow DOM, as discussed in [issue 909](https://github.com/WICG/webcomponents/issues/909#issuecomment-1977487651). While this scenario is related to sharing styles with Shadow DOM elements, it is solving a different problem and should be addressed separately.
 
-## Use case     
+## Use case    
+### Pages with sub-sections
```

Comment:
I'm wondering if we should update the example to something similar to what is seen in the wild. Potentially a fake page with a table of contents section on the right or left?

Reply:
I think an additional sentence like this will suffice, what do you think? "This pattern of pages with sub-sections are often seen in blogs, search result pages, pages that has a separate table of content, etc".

Reply:
My concern was less about the text but more about the example used. I'm thinking we can showcase something that is more realistic to patterns we see in the wild. I think a table of contents of some kind would be more identifiable to someone reading the explainer